zonemaster/zonemaster-gui
# Update Danish translation (FAQ mostly)

## Purpose

Try to fix the missing translations of the FAQ.

## Context

The translations system of the FAQ has changed. This import old translations into the new system.

## Changes

Danish translations.

## How to test this PR

A quick check of the translated string would be welcome to verify that I haven't done any mistake while copy/pasting.

**Note**: There are some missing translation for the strings that had changed since the last FAQ translation. Also some may have drifted a bit so feel free to update where the translation could be improved to be more in line with the original text.
